src/script/cpp_api/s_player.cpp

```diff
@@ -109,9 +109,9 @@ s32 ScriptApiPlayer::on_player_hpchange(ServerActiveObject *player,
 
 	// Call callbacks
 	PCALL_RES(lua_pcall(L, 3, 1, error_handler));
-	hp_change = lua_tointeger(L, -1);
+	lua_Integer change = lua_tointeger(L, -1);
```

Comment:
`s32` is already an int, so is `lua_Integer`
just don't

Reply:
Alright, let it be s32